x-pack/plugins/cloud_security_posture/public/pages/benchmarks/benchmarks_table.tsx

```diff
@@ -11,66 +11,111 @@ import {
   type EuiBasicTableProps,
   type Pagination,
   type CriteriaWithPagination,
-  EuiLink,
-  EuiToolTip,
-  EuiAvatar,
   EuiEmptyPrompt,
+  EuiFlexGroup,
+  EuiFlexItem,
 } from '@elastic/eui';
 import React from 'react';
-import { generatePath } from 'react-router-dom';
-import { pagePathGetters } from '@kbn/fleet-plugin/public';
 import { i18n } from '@kbn/i18n';
-import type { PackagePolicy } from '@kbn/fleet-plugin/common';
 import { FormattedMessage } from '@kbn/i18n-react';
-import { TimestampTableCell } from '../../components/timestamp_table_cell';
-import type { Benchmark } from '../../../common/types';
-import { useKibana } from '../../common/hooks/use_kibana';
-import { benchmarksNavigation } from '../../common/navigation/constants';
+
+import type { BenchmarkVersion2, BenchmarkScore } from '../../../common/types';
 import * as TEST_SUBJ from './test_subjects';
-import { getEnabledCspIntegrationDetails } from '../../common/utils/get_enabled_csp_integration_details';
 import { isCommonError } from '../../components/cloud_posture_page';
 import { FullSizeCenteredPage } from '../../components/full_size_centered_page';
+import { ComplianceScoreBar } from '../../components/compliance_score_bar';
+import { getBenchmarkCisName, getBenchmarkApplicableTo } from '../../../common/utils/helpers';
+import { CISBenchmarkIcon } from '../../components/cis_benchmark_icon';
 
 export const ERROR_STATE_TEST_SUBJECT = 'benchmark_page_error';
 
 interface BenchmarksTableProps
-  extends Pick<EuiBasicTableProps<Benchmark>, 'loading' | 'error' | 'noItemsMessage' | 'sorting'>,
+  extends Pick<
+      EuiBasicTableProps<BenchmarkVersion2>,
+      'loading' | 'error' | 'noItemsMessage' | 'sorting'
+    >,
     Pagination {
-  benchmarks: Benchmark[];
-  setQuery(pagination: CriteriaWithPagination<Benchmark>): void;
+  benchmarks: BenchmarkVersion2[];
+  setQuery(pagination: CriteriaWithPagination<BenchmarkVersion2>): void;
   'data-test-subj'?: string;
 }
 
-const AgentPolicyButtonLink = ({ name, id: policyId }: { name: string; id: string }) => {
-  const { http } = useKibana().services;
-  const [fleetBase, path] = pagePathGetters.policy_details({ policyId });
+// Commented Out until the full table is made
```

Comment:
is it a leftover for this PR or smth that will be introduced in the follow-up? should we delete this comment and just introduce it in the follow-up? if you don't want to loose the code, you can always check the change history to get it back

Reply:
yep, this will be introduced back when we can access the rules page again ( I have a separate PR for that ), but I can just remove this for now